### ISSUE TITLE
Remove the Photon card from the JP dashboard

### ISSUE THREAD

**Issue by MichaelArestad** [Automattic/jetpack#6978]
Right now I don't think there is a great reason to keep the Photon card in the dashboard.

![image](https://cloud.githubusercontent.com/assets/1123119/25054814/7a0ffc3a-211c-11e7-9318-8d79104b26c5.png)

It's the sole holdout in the Performance section.

CC @jeffgolenski @rickybanister @richardmuscat

**Comment by richardmuscat** [comment]
I don't disagree that its weak (and also the Protect one: #6979), but before we remove stuff that people may have gotten used to, I'd much prefer seeing a broader plan for where we're going with the dashboard rather than simply remove/add bits at (seemingly) random.

The intention behind having these cards on the dashboard was to show Jetpack users the ongoing, often under-the-hood value of Jetpack that they might not be aware of and/or have forgotten about.

I don't think we should remove them simply because they're not helpful/inspiring enough (taking your comment from #6979) until we have an idea of how to make the dashboard as a whole more helpful/inspiring.

While they may not be super useful, they serve a purpose which will be neglected if we just removed them.

**Comment by rickybanister** [comment]
I agree, can we do some exploration about exposing photon data via an endpoint? Can we tell them how much media they've uploaded? If they have VideoPress active we should perhaps make a combined 'CDN' card here showing the total value in bandwidth savings and global accessibility. Not that I would use the term 'CDN' anywhere.

**Comment by MichaelArestad** [comment]
Since creating this, we have pivoted and perfomance is a focus. Instead of removing this, perhaps we figure out how to make it valuable.

**Comment by jeherve** [comment]
The card is gone now. closing this issue.
